internal/services/apimanagement/api_management_logger_resource_test.go

```diff
@@ -61,6 +61,31 @@ func TestAccApiManagementLogger_requiresImport(t *testing.T) {
 	})
 }
 
+func TestAccApiManagementLogger_managedIdentityEventHub(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_api_management_logger", "test")
+	r := ApiManagementLoggerResource{}
+
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.managedIdentityEventHub(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+				check.That(data.ResourceName).Key("buffered").HasValue("true"),
+				check.That(data.ResourceName).Key("eventhub.#").HasValue("1"),
+				check.That(data.ResourceName).Key("eventhub.0.name").Exists(),
+				check.That(data.ResourceName).Key("eventhub.0.endpoint_address").Exists(),
+				check.That(data.ResourceName).Key("eventhub.0.client_id").HasValue("SystemAssigned"),
+			),
+		},
+		{
+			ResourceName:            data.ResourceName,
+			ImportState:             true,
+			ImportStateVerify:       true,
+			ImportStateVerifyIgnore: []string{"eventhub.0.endpoint_address", "eventhub.0.client_id"},
+		},
```

Comment:
This can be simplified 
```suggestion
		data.ImportStep(),
```